packages/common-css/README.md

```diff
@@ -1,3 +1,37 @@
-# Fundamental Library Common CSS Package
+Fundamental Library Common CSS Package
 
-This package contains Common CSS files. More documentation to come soon...
+## What is Fundamental Library Common CSS?
+
+Fundamental Library Common CSS Package holds all the CSS style rules of the library that are recognized by older browsers—as well as the newer browsers. It creates the style rules for the basic elements on a page, such as a (link) elements, p (paragraph) elements, h1 (header) elements etc. The package is based on the DRY (Don't Repeat Yourself) methodology which allows the developers to not repeat the same CSS rules overall in the library.
+
+## Getting Started
+
+### 1. Fundamental Library Common CSS Package included in Fundamental Styles
+
+Fundamental Library Common CSS Package is part of the Fundamental Styles. If you have installed the Fundamental Styles in your project, you automatically get the Fundamental Library Common CSS Package.
+
+**Learn more at http://sap.github.io/fundamental-styles/**
+
+### 2. Fundamental Library Common CSS Package on its own
+
+Fundamental Library Common CSS Package can stand on its own. You can install the package in your project through:
+
+```bash
+npm i @fundamental-styles/common-css
+```
+
+## Common CSS Elements
+
+The Fundamental Library Common CSS Package consists of elements that align with the SAP designs.
```

Comment:
```suggestion
## Common CSS Properties

The Fundamental Library Common CSS Package consists of property values that align with the SAP designs.
```

Reply:
I think here we should mention it's SAP Fiori design